core/src/test/scala/unit/kafka/server/MetadataCacheTest.scala

```diff
@@ -511,6 +512,101 @@ class MetadataCacheTest {
     assertEquals(initialBrokerIds.toSet, aliveBrokersFromCache.map(_.id).toSet)
   }
 
+  @ParameterizedTest
+  @MethodSource(Array("cacheProvider"))
+  def testGetPartitionReplicaEndpoints(cache: MetadataCache): Unit = {
```

Comment:
Does this mean that we didn't have a test for this method?

Reply:
Mostly - there is an FFF integration test verifying that the method doesn't return a shutdown node (eventually).